app/views/state_file/questions/federal_extension_payments/edit.html.erb

```diff
@@ -1,11 +1,12 @@
-<% title = t(".title")%>
+<% i18n_scope = [:state_file, :questions, :federal_extension_payments, :edit, current_state_code] %>
+<% title = I18n.t(:title, scope: i18n_scope, default: t(".title")) %>
 
 <% content_for :page_title, title %>
 <% content_for :card do %>
   <div class="spacing-above-25">
     <h1 class="h2"><%= title %></h1>
-    <p><%= t(".subtitle_1") %></p>
-    <p><%= t(".subtitle_2") %></p>
+    <p><%= I18n.t(:subtitle_1, scope: i18n_scope, default: t(".subtitle_1")) %></p>
+    <p><%= I18n.t(:subtitle_2, scope: i18n_scope, default: t(".subtitle_2")) %></p>
```

Comment:
why do these need to be az-specific? isn't it the same copy from state to state?